<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintSetupDialog</class>
 <widget class="QDialog" name="PrintSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrintSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbPrinter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Printer</string>
   </property>
   <widget class="QLabel" name="lbPrinter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Printer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPrinter">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbReport">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>Report</string>
   </property>
   <widget class="QLabel" name="lbUsbReport">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usb printer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbUsbReport">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPrintGraph">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Print graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbThermalReport">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thermal printer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbThermalReport">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
